app/models/ability.rb

```diff
@@ -45,6 +45,11 @@ def not_signed_in
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can [:show, :create], User
+
+    can [:new, :create], Event do |event|
```

Comment:
For the not_signed_in user this includes user creation, which we don't want to have with ichain (right?).